authors/apps/articles/views.py

```diff
@@ -97,3 +97,56 @@ def destroy(self, request, slug):
             {"message": "Article is deleted"},
             status=status.HTTP_200_OK
         )
+
+
+class LikeArticleAPIView(generics.UpdateAPIView):
+    permission_classes = (IsAuthenticated,)
+    queryset = ArticleLikes.objects.all()
+
+    def get_object(self):
+        """Method to get an article"""
+        slug = self.kwargs["slug"]
```

Comment:
@VivianDoreen The PO talked about this, and said we should consider using `kwargs.get('slug')` which is safer since it returns None in case the slug isn't found other than `kwargs['slug']` which will throw a "key error"